legend-engine-authentication-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/authentication/AuthenticationSpec.java

```diff
@@ -0,0 +1,18 @@
+package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.authentication;
```

Comment:
package path seems a bit off

Reply:
Refactored to org.finos.legend.engine.protocol.pure.v1.model.packageableElement.runtime.connection.authentication